### PR TITLE
Preparing release 1.11

### LINKED ISSUE #1343: Release 1.11 scope
* [ ] make dependencies_v2 the default for Cassandra #1344
* [x] make gRPC reporter agent-collector default https://github.com/jaegertracing/jaeger/issues/1218
* [x] ES support TLS with credentials https://github.com/jaegertracing/jaeger/issues/1327

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #1343 

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,10 +45,20 @@ Changes by Version
     ```
 ##### New Features
 
+- Support secure gRPC channel between agent and collector ([#1391](https://github.com/jaegertracing/jaeger/pull/1391), [@ghouscht](https://github.com/ghouscht), [@yurishkuro](https://github.com/yurishkuro))
+
 ##### Bug fixes, Minor Improvements
 
+- Allow thrift reporter even if grpc hosts are not provided ([#1400](https://github.com/jaegertracing/jaeger/pull/1400), [@pavolloffay](https://github.com/pavolloffay))
+- Allow to use TLS with ES basic auth ([#1388](https://github.com/jaegertracing/jaeger/pull/1388), [@pavolloffay](https://github.com/pavolloffay))
```
Comment (yurishkuro) [inline CHANGELOG.md]: feels like new feature

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,10 +45,20 @@ Changes by Version
...
+- Deprecate colon in index prefix in ES dependency store ([#1386](https://github.com/jaegertracing/jaeger/pull/1386), [@pavolloffay](https://github.com/pavolloffay))
+- Make grpc reporter default and add retry ([#1384](https://github.com/jaegertracing/jaeger/pull/1384), [@pavolloffay](https://github.com/pavolloffay))
```
Comment (yurishkuro) [inline CHANGELOG.md]: does this not qualify as breaking change?

File: CHANGELOG.md [hunk 3]
```diff
@@ -31,10 +45,20 @@ Changes by Version
     ```
 ##### New Features
 
+- Support secure gRPC channel between agent and collector ([#1391](https://github.com/jaegertracing/jaeger/pull/1391), [@ghouscht](https://github.com/ghouscht), [@yurishkuro](https://github.com/yurishkuro))
+
 ##### Bug fixes, Minor Improvements
 
+- Allow thrift reporter even if grpc hosts are not provided ([#1400](https://github.com/jaegertracing/jaeger/pull/1400), [@pavolloffay](https://github.com/pavolloffay))
```
Comment (annanay25) [inline CHANGELOG.md]: nit: Should this be - Allow thrift reporter ~~even~~ if grpc hosts are not provided

File: CHANGELOG.md (hunk 1 above)
Comment (pavolloffay) [reply CHANGELOG.md]: I have classified it as minor improvement :) I can move it to new features section though

File: CHANGELOG.md (hunk 2 above)
Comment (pavolloffay) [reply CHANGELOG.md]: No bc of https://github.com/jaegertracing/jaeger/pull/1400

Comment (pavolloffay) [reply CHANGELOG.md]: we still support old flags

**Pull request merged**